src/config/dynamic-theme-fixes.config

```diff
@@ -16690,6 +16690,9 @@ CSS
 html[hide-scrollbar] ::-webkit-scrollbar {
     display: none !important;
 }
+#notification-count.ytd-notification-topbar-button-renderer {
```

Comment:
The notification border is on the original site, so we shouldn't be removing it.